src/Lottery.js

```diff
@@ -0,0 +1,47 @@
+const { getArrayToCustomMessage, pickUniqueNumbersInRange, print } = require('./lib/Utils');
+const { LOTTO_MAX_NUMBER, LOTTO_MIN_NUMBER, LOTTO_NUMBER } = require('./lib/Constants');
+
+class Lottery {
+  #lottoList = [];
+
+  lottoCount = 0;
+
+  constructor(lottoCount) {
+    this.lottoCount = lottoCount;
+  }
+
+  pick() {
+    const lottos = [];
+
+    for (let index = 0; index < this.lottoCount; index += 1) {
+      const lotto = pickUniqueNumbersInRange(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER, LOTTO_NUMBER);
+      lottos.push(lotto.sort((a, b) => a - b));
+    }
```

Comment:
```suggestion
    const lottos = Array.from({ length: lottoCount }, () => pickUniqueNumbersInRange(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER, LOTTO_NUMBER));
    return lotto.sort((a, b) => a - b);
```
로도 바꿀 수 있을 것 같습니다..!